src/main/java/org/skife/jdbi/v2/BeanMapper.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2004 - 2014 Brian McCallister
+ * Copyright (C) 2004 - 2015 Brian McCallister and others
```

Comment:
I take that to mean that every file touched in this PR should have updated license text?